azure-kusto-data/setup.py

```diff
@@ -41,6 +41,6 @@
     namespace_packages=["azure"],
     keywords="kusto wrapper client library",
     packages=find_packages(exclude=["azure", "tests"]),
-    install_requires=["adal>=1.0.0", "python-dateutil>=2.8.0", "requests>=2.13.0", "msrestazure>=0.4.14"],
+    install_requires=["adal>=1.0.0", "python-dateutil>=2.8.0", "requests>=2.13.0", "msrestazure>=0.4.14", "aiohttp==3.6.2", "asgiref==3.2.3", "aiofile==1.5.2"],
```

Comment:
these should be optionals. (like the pandas thing). people using sync flow don't need them.

Reply:
Ammm... I am not sure I am following you with this? why are they optional?

Reply:
In theory, installing these packages takes time and space.
TBH, I'm still not sure I want async flow to be the default right now.

I want people to **opt-in** into this flow.
This means that people that want to use the regular flow, still can without extra cost.